cluster/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/cluster/typed/ClusterShardingTypedModule.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.internal.javadsl.cluster.typed
+
+import akka.actor.ActorSystem
+import akka.cluster.sharding.typed.javadsl.ClusterSharding
+import javax.inject.Inject
+import javax.inject.Provider
+import play.api.Configuration
+import play.api.Environment
+import play.api.inject.Binding
+import play.api.inject.Module
+
+class ClusterShardingTypedModule extends Module {
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = Seq(
+    bind[ClusterSharding].toProvider[ClusterShardingTypedProvider]
```

Comment:
Only exposes `ClusterSharding`. Neither `Cluster` (typed) nor `ActorSystem[_]` are exposed. This should make the migration of this new `Module` to Play easier.